gobblin-metrics-libs/gobblin-metrics/src/main/java/org/apache/gobblin/metrics/ServiceMetricNames.java

```diff
@@ -65,4 +65,8 @@ public class ServiceMetricNames {
   public static final String HELIX_LEADER_STATE = "HelixLeaderState";
 
   public static final String FLOWGRAPH_UPDATE_FAILED_METER = GOBBLIN_SERVICE_PREFIX + ".FlowgraphUpdateFailed";
+
+  public static final String DAG_COUNT_MYSQL_DAG_STATE_COUNT = GOBBLIN_SERVICE_PREFIX + ".MysqlDagStateStore" + ".totalDagCount";
+
+  public static final String DAG_COUNT_FS_DAG_STATE_COUNT = GOBBLIN_SERVICE_PREFIX + ".FsDagStateStore" + ".totalDagCount";
```

Comment:
do we need these to have separate names? what if we have it as one metric and we can distinguish which store is used from configs deployed? how do we handle this for other cases like the `quotaManager` where we have multiple implementations that could be used?

Reply:
I believe having separate names would help for better tracking, especially to understand how many `FSDagStateStore`s are still in use. Thus, the idea is separate it at individual impl. level to track the metrics better, rather having to combine them... and for cases like `quotaManager` might not arise here since we would have only impl. used at any given time per service deployment